modules/local/bwa_aln_to_bam/main.nf

```diff
@@ -0,0 +1,61 @@
+process BWA_ALN_TO_BAM {
```

Comment:
why are you using the local module and the nf-core one? they look the same to me

Reply:
replaced with the fastq_align_bwa subworkflow